medusa/name_parser/rules/rules.py

```diff
@@ -122,6 +122,11 @@ def when(self, matches, context):
             if not group or matches.at_match(group):
                 continue
 
+            # don't use websites as release group
```

Comment:
Will this change prevent these guesses?
```
# guessit: 3.0.0  rebulk: 0.9.0
? [TorrentCouch.net].Mr.Mercedes.S02E09.WEB.x264.[236MB].mp4
: release_group: TorrentCouch
  title: Mr Mercedes
  season: 2
  episode: 9
  source: Web
  video_codec: H.264
  video_encoder: x264
  size: 236MB
  container: mp4
  mimetype: video/mp4
  type: episode
  parsing_time: 0.688663005829
```
Asking because `TorrentCouch` in here is an acceptable guess.

Reply:
No, this rule only applies if there is a group at the end, like: `[TorrentCouch.net].Mr.Mercedes.S02E09.WEB.x264.[236MB]-GROUP.mp4`
Without the change it would mistakenly use `TorrentCouch` as the group.